release/amd64/make-memstick.sh

```diff
@@ -60,12 +60,10 @@ fi
 
 # Make an ESP in a file.
 espfilename=$(mktemp /tmp/efiboot.XXXXXX)
-if [ -f "${BASEBITSDIR}/boot/loader_ia32.efi" ]; then
-	make_esp_file ${espfilename} ${fat32min} ${BASEBITSDIR}/boot/loader.efi bootx64 \
-	    ${BASEBITSDIR}/boot/loader_ia32.efi bootia32
-else
-	make_esp_file ${espfilename} ${fat32min} ${BASEBITSDIR}/boot/loader.efi
+if [ ${MK_LOADER_IA32} = "yes" ]; then
+	extra_args="${BASEBITSDIR}/boot/loader_ia32.efi bootia32"
```

Comment:
Should "bootx64" be in the extra_args case here?  It's not present in the existing code for the !LOADER_IA32 case.

Reply:
It shouldn't matter. If `make_esp_file` isn't given the "bootx64", it automatically determines it using `uname -m`.
I just thought this would make more sense since the arguments are given in pairs, and the condition was for the ia32 pair.